### ISSUE TITLE
nvda development under windows uneversal'nuju

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7017]
whether the accessibility of nvda as a universal application in a future Microsoft releases windows Edition 10 installation will be allowed only where the universal applications. NVDA takes a lot of people and I believe that in the future this will come in handy.

**Comment by josephsl** [comment]
Hi,

So are you asking that NVDA become a universal app? This cannot be done for now due to technical reasons. We'll reconsider it once the Universal Windows Platform matures and a possibility to create universla NVDA app becomes feasible. Thanks.

**Comment by feerrenrut** [comment]
I'm going to close this issue. As @josephsl has pointed out, for now we are not able to make NVDA a universal app.

**Comment by josephsl** [comment]
Hi,

Coming back to this...

Although NVDA cannot become a full universal app, we now have foundations to convert it into a hosted application. Still, this issue will remain closed because of the fact that this is resolved with AppX checks and because according to @MichaelDCurran, NVDA cannot be listed on Microsoft Store yet due to bugs from Microsoft side.

Thanks.
